src/pybind/mgr/dashboard/frontend/cypress/integration/orchestrator/01-hosts.e2e-spec.ts

```diff
@@ -82,5 +82,10 @@ describe('Hosts page', () => {
       const hostname = Cypress._.sample(this.hosts).name;
       hosts.maintenance(hostname, true);
     });
+
+    it('should exit host from maintenance', function () {
+      const hostname = Cypress._.sample(this.hosts).name;
+      hosts.checkServiceInstancesExist(hostname, ['mgr: 1', 'prometheus: 1']);
```

Comment:
hmm i am not sure if we can expect the prometheus to be in the teuthology branch. normally it is started without monitoring stacks.

Reply:
Yeah, I'm testing with cephadm e2e now instead.